wagtail/admin/views/pages/edit.py

```diff
@@ -55,6 +56,45 @@ def add_save_confirmation_message(self):
 
         messages.success(self.request, message)
 
+    def send_commenting_notifications(self):
+        """
+        Sends notifications about any changes to comments to anyone who is subscribed.
+        """
+        # Skip if this page does not have CommentPanel enabled
+        if 'comments' not in self.form.formsets:
+            return
+
+        # Get changes
+        comments_formset = self.form.formsets['comments']
+        new_comments = comments_formset.new_objects
+        deleted_comments = comments_formset.deleted_objects
+
+        # Assume any changed comments that are resolved were only just resolved
+        resolved_comments = []
+        for changed_comment, changed_fields in comments_formset.changed_objects:
+            if changed_comment.resolved_at and 'resolved' in changed_fields:
+                resolved_comments.append(changed_comment)
```

Comment:
This doesn't include replies as far as I can see - new replies are also something people should be notified about, and I believe the idea was that even with "page level" notifications off, anyone in the thread would get reply notifications.